packages/coding-standard/src/Rules/NoScalarAndArrayConstructorParameterRule.php

```diff
@@ -95,12 +95,10 @@ private function isValueObjectNamespace(Class_ $class): bool
     private function getPossibleTypes($type): array
```

Comment:
I think here should be some type check, in param type

Reply:
I can handle it, if it's too much struggle :+1:

Reply:
thank you :+1:

Reply:
No troubles :+1: